Makefile

```diff
@@ -157,12 +157,18 @@ fmt: ## Check formatting of go sources.
 	@$(MAKE) go.init
 	@$(MAKE) go.fmt
 
+.PHONY: markdownlint
+# the markdownlint  target requires
+# markdownlint-cli2 to be available locally in the PATH
```

Comment:
This isn't documented for pylint, so I don't see a need to doc that here

Reply:
agreed. removed.